libs/template/presets/storefront/experience/pages/registration-page.ts

```diff
@@ -0,0 +1,35 @@
+import { ExperienceComponent } from '@spryker-oryx/experience';
```

Comment:
Shouldn't we also add a ling to this page somewhere? Or I missed it?

Reply:
Added the button on Login page. Also, aligned the Registration page with designs to make user able to move back to Login page